src/OptinForms/AbstractOptinTheme.php

```diff
@@ -1049,7 +1049,7 @@ public function shortcode_optin_form_custom_fields($atts)
 
                     $options = [];
                     if ( ! empty($field['field_options'])) {
-                        $options = array_map('trim', explode(',', $field['field_options']));
+                        $options = preg_split( "/\\r\\n|\\r|\\n/", $field['field_options']);
```

Comment:
@ShoboySnr hey.. looks like you do not think of old or existing users when making changes. This is not good

what happens to customer already using comma to separate the options?

To maintain backward compatibility, I prefer something like

Name|f_name, Email|email

If no pipe is found, default the entered data to be the option name and value. Just like the old behaviour.

Reply:
Ohhh.. Okay. Got it. I'll effect the change immediately